Explanation:  FreeBSD/ARM has a bunch of numeric support routines
(double-precision mod, for example) in libc instead of libcompiler_rt
or libgcc.  So freestanding programs have to be linked against libc.

diff --git a/Makefile b/Makefile
index 99b9278..8395284 100644
--- a/Makefile
+++ b/Makefile
@@ -312,7 +312,7 @@ endif
 else
 PLATFORM_LIBGCC = -L $(shell dirname `$(CC) $(CFLAGS) -print-libgcc-file-name`) -lgcc
 endif
-PLATFORM_LIBS += $(PLATFORM_LIBGCC)
+PLATFORM_LIBS += $(PLATFORM_LIBGCC) -lc
 export PLATFORM_LIBS
 
 # Special flags for CPP when processing the linker script.
